sample-compose-fragments/src/main/res/drawable-v24/ic_launcher_foreground.xml

```diff
@@ -0,0 +1,30 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can we copy over the app icon from the existing component sample app to use as the launcher icon, and put it a different colour background (maybe just white) so it's identifiable as our component library, but different to the existing one?